packages/bierzo-wallet/src/components/wallet/Wallet.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```

Comment:
this can be renamed to `index.tsx`. All default/main components should be named in this way.

Reply:
Fixed by [14f6201](https://github.com/iov-one/ponferrada/pull/79/commits/14f6201854bd623d3fb74608509f7e5b7c587dc8)